pygeohash/nbgeohash.py

```diff
@@ -8,15 +8,13 @@
 
 """
 
-
-__base32 = "0123456789bcdefghjkmnpqrstuvwxyz"
-#  Note: the alphabet in geohash differs from the common base32 alphabet described in IETF's RFC 4648
-# (http://tools.ietf.org/html/rfc4648)
```

Comment:
I didn't see a good reason for re-declaration of this constant, but in theory if someone downstream is doing `from pygeohash.nbgeohash import __base32` this *might* break that? I'd have to check.